panel.go

```diff
@@ -567,6 +567,11 @@ type Target struct {
 	PerSeriesAligner   string                    `json:"perSeriesAligner,omitempty"`
 	ValueType          string                    `json:"valueType,omitempty"`
 	GroupBys           []string                  `json:"groupBys,omitempty"`
+	Tags               []struct {
+		Key      string `json:key,omitempty`
```

Comment:
All of these are `key`? :scream: doesn't look correct. Also, could we add tests for this? From which Grafana version is this available?

Reply:
oops, yes, that's a failure on my part. i'll fix that.

Reply:
Tags on targets are available from at least Grafana 6

Reply:
Could we please add an integration test for this, then? It would be very helpful

Reply:
do you have a preference for adding these anywhere? I don't see a `Panel` integration test file, but I do for dashboards.